index.php

```diff
@@ -483,6 +486,19 @@ function smoothScrollToTop() {
         }
     });
 
+    document.addEventListener("DOMContentLoaded", function() {
+        if (isLoggedIn) {
+            document.querySelector('.logout-btn').style.display = 'block';
+            document.querySelector('.login-btn').style.display = 'none';
+            document.querySelector('.register-btn').style.display = 'none';
+        } else {
+            document.querySelector('.logout-btn').style.display = 'none';
+        }
+    });
```

Comment:
<div>


<div id="suggestion">
<div id="issue"><b>Consider extracting DOM manipulation logic</b></div>
<div id="fix">

Consider moving the DOM manipulation logic for login/logout buttons into a separate function for better code organization and reusability. The `DOMContentLoaded` event listener could be simplified by calling this function.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<blockquote>Check the AI-generated fix before applying</blockquote>
<div id="code">


```suggestion
    function updateAuthButtons() {
        if (isLoggedIn) {
            document.querySelector('.logout-btn').style.display = 'block';
            document.querySelector('.login-btn').style.display = 'none';
            document.querySelector('.register-btn').style.display = 'none';
        } else {
            document.querySelector('.logout-btn').style.display = 'none';
        }
    }

    document.addEventListener("DOMContentLoaded", function() {
        updateAuthButtons();
    });
```

</div>
</details>
</div>


<small><i>Code Review Run <a href=https://github.com/Lux-Jason/rms_new/pull/6#issuecomment-2543456910>#da6ca6</a></i></small>
</div>

---
Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged